src/core/bau_inputs.jl

```diff
@@ -162,6 +162,9 @@ function BAUInputs(p::REoptInputs)
     heating_loads_served_by_tes = Dict{String,Array{String,1}}()
     unavailability = get_unavailability_by_tech(p.s, techs, p.time_steps)
 
+    # Calculate clean energy contribution (kW)
+    calculate_clean_energy_contribution(p.s.electric_utility, p.s.electric_load)
```

Comment:
@pypapus I think you'll want to use the `bau_grid_to_load` variable here instead of p.s.electric_load. This is because `p.s.electric_load` doesn't account for existing PV. The calc can mirror the bau_grid_emissions_lb_CO2_per_year calculation